includes/template-functions.php

```diff
@@ -26,7 +26,9 @@ function course_single_lessons() {
 
 	}
 
-	Sensei_Templates::get_template( 'single-course/lessons.php' );
+	$lessons_query = Sensei_Course::get_single_course_lessons_query();
```

Comment:
We could probably move this inside `lessons.php`

Reply:
Do you mean keep here only the `Sensei_Templates::get_template( 'single-course/lessons.php' );`.
And inner the `sensei/templates/single-course/lessons.php` the following code?

```php
$lessons_query = Sensei_Course::get_single_course_lessons_query();
wp_reset_postdata();
```

If yes, I wonder if it would be good to keep the current code to keep the logic out of the template. Honestly, I'm not sure, but if you really think it's better to move the code to the template, it's okay for me. Wait for your answer after my justification to see if I move or not. 😊

Reply:
I see your point of keeping the template clean. Lets leave it as it is.